lib/src/Base/Algo/openturns/LeastSquaresMethodImplementation.hxx

```diff
@@ -94,6 +94,9 @@ public:
   /** Get the diagonal of H */
   virtual NumericalPoint getHDiag() const;
 
+  /** Get the diagonal of the Gram inverse */
+  virtual NumericalPoint getGramInverseDiag() const;
```

Comment:
getGramInverseDiag is not documented (neither are getGramInverse(), getGramInverseTrace() getH() 	getHDiag() but that's my bad)